scss-fashionz/scss/variables/_variables.scss

```diff
@@ -0,0 +1,37 @@
+
+
+
+$primary-bg-color : #2a2b30;
+
+$bg-light : #d8d8d8;
+
+$prod-bg-image : #24252a;
+
+$font-semibold: 600;
+
+$border-light : #949494;
+
+$text-light : #949494;
+
+$text-white : #fff;
+
+$text-dark : #000;
+
+$mb-0_5 : 0.5rem;
+
+$m-1rem : 1rem;
+
+$mb-1rem : 1rem;
+
+$text-green : #88b282;
```

Comment:
Would be better to have comments and group the variables. as we discussed yesterday